deps/lua/src/lstring.c

```diff
@@ -71,14 +72,55 @@ static TString *newlstr (lua_State *L, const char *str, size_t l,
   return ts;
 }
 
+uint32_t murmur32(const uint8_t* key, size_t len, uint32_t seed) {
+  static const uint32_t c1 = 0xcc9e2d51;
+  static const uint32_t c2 = 0x1b873593;
+  static const uint32_t r1 = 15;
+  static const uint32_t r2 = 13;
+  static const uint32_t m = 5;
+  static const uint32_t n = 0xe6546b64;
+  uint32_t hash = seed; // static seed 
```

Comment:
```suggestion
  uint32_t hash = seed;
```
The comment is a bit confusing since the seed is not static.